.github/workflows/apply.yml

```diff
@@ -7,48 +7,54 @@ on:
         description: The environment to deploy against.
         type: string
         default: intg
+      project:
+        required: true
+        description: The project to run this for
+        type: choice
+        options:
+          - tdr
+          - dr2
 permissions:
   id-token: write
   contents: write
 jobs:
+  setup:
+    uses: nationalarchives/dr2-github-actions/.github/workflows/set_terraform_var_names.yml@main
+    with:
+      project: ${{ github.event.inputs.project }}
+      environment: ${{ github.event.inputs.environment }}
   apply:
-    uses: nationalarchives/tdr-github-actions/.github/workflows/terraform_apply.yml@main
+    uses: nationalarchives/dr2-github-actions/.github/workflows/terraform_apply.yml@main
+    needs: setup
     with:
-      repo-name: tdr-aws-accounts
+      repo-name: tna-custodian
```

Comment:
Should the `repo-name` be `tna-custodian`?

Reply:
Good spot, cheers. Copy and paste error.